src/sentry/migrations/0954_add_group_by_to_snuba_query.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 5.2.1 on 2025-07-23 17:29
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0953_make_releasefiles_tti"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="snubaquery",
+            name="group_by",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.CharField(max_length=1000), null=True, size=100
```

Comment:
100 seems like sanity cap, will probably be a lot lower in practice. also, 1000 was the largest "length" value found for what our attributes can be.